### PR TITLE
Appointment time issue fix

### PR DESCRIPTION
**Pull request by GourRohit**
Fixed the appointment time issue.

### CODE REVIEW

File: vds-backend/index.js [hunk 1]
```diff
@@ -102,28 +102,21 @@ function checkForDuplicateData(docNumber) {
 app.post("/data", async (req, res) => {
   // Returns the current epoch time in milliseconds
   const currentEpochTime = Date.now();
-  // Create a new Date object with the epoch time
-  const date = new Date(currentEpochTime);
-  // Add 30 minutes to the Date object
```
Comment (naga-cid) [inline vds-backend/index.js:107]: why is this code needed?

File: vds-backend/index.js [hunk 2]
```diff
@@ -102,28 +102,21 @@ function checkForDuplicateData(docNumber) {
...
-  };
   if (!rows.length > 0) {
     await insertData(data);
     message = "success";
-    appointmentTime = convertEpochToTime(data.appointmentTime);
+    appointmentTime = data.appointmentTime;
   } else {
     message = "duplicate";
-    appointmentTime = convertEpochToTime(rows[0].appointmentTime);
+    appointmentTime = Number.parseInt(rows[0].appointmentTime);
```
Comment (naga-cid) [inline vds-backend/index.js:119]: you dont need this parseint

File: vds-backend/index.js (hunk 1 above)
Comment (GourRohit) [reply vds-backend/index.js]: Yes, its not needed.

File: vds-backend/index.js (hunk 2 above)
Comment (GourRohit) [reply vds-backend/index.js]: Yes, I have removed it now.

**Review by naga-cid — APPROVED** [review]

**Pull request merged**